pkg/middleware/requestid/requestid.go

```diff
@@ -46,13 +48,13 @@ func reportable() interceptors.CommonReportableFunc {
 		id, _ := uuid.NewRandom()
 		requestID := id.String()
 
-		// Add the requestID to the context
+		// Add the requestID to the context.
```

Comment:
This comment and the ones immediately following go without saying. The comments don't add much value. I'd recommend removing them TBH.